ArcGISRuntimeSDKQt_QMLSamples/Maps/DownloadPreplannedMap/DownloadPreplannedMap.qml

```diff
@@ -0,0 +1,284 @@
+// [WriteFile Name=DownloadPreplannedMap, Category=Maps]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import Esri.ArcGISRuntime 100.7
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url outputMapPackage: System.temporaryFolder.url + "/ArcGIS"
+    property var preplannedMapAreaList: null
+    property var preplannedArea: null
+    property var path: null
+    property var mapExists: false
+    property var viewingOnlineMaps: true
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            id: onlineMap
+            item: portalItem
+        }
+
+        Portal {
```

Comment:
I don't think you need to declare a `Portal` if it's the default `arcgis.com`. You could also move the portal and the item outside of the MapView scope